src/components/Map.tsx

```diff
@@ -69,15 +86,23 @@ const MapContainer = () => {
 
   useEffect(() => {
     const url = new URL(window.location.href);
-    if (url.searchParams.has("from") && url.searchParams.has("to")) {
-      const from = parseLngLat(url.searchParams.get("from")!);
-      const to = parseLngLat(url.searchParams.get("to")!);
-      getQuery(from, to);
-    } else if (url.searchParams.has("from")) {
-      const from = parseLngLat(url.searchParams.get("from")!);
-      updateQueryFromParam(from);
+    if (props.isWidget) {
+      if (props.isWidget && url.searchParams.has("from") && dest) {
+        const from = parseLngLat(url.searchParams.get("from")!);
+        getQuery(from, dest);
+      } else if (url.searchParams.has("from")) {
```

Comment:
Is this a valid case in widget mode? That there is no destionation set?

Reply:
Hmm, I guess not. Removing it.